oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCUserManager.java

```diff
@@ -204,14 +204,14 @@ private int sendLogout()
         try {
             logoutURI = this.configuration.getLogoutOIDCEndpoint();
         } catch (Exception e) {
-            // TODO: log something ?
+            this.logger.error("The String could not be parsed as a URI reference", e);
```

Comment:
I would suggest something a little more generic like "Failed to generate the logout endpoint URI".

Reply:
Done.